docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+# start bitcoin backend
+cd /opt/coins/nodes/bitcoin_regtest
```

Comment:
I am noticing that quite late, but are these two `cd`'s necessary? Seems all the filenames we use are absolute, so our location does not matter.

If they are really redundant, we can choose if we remove them or make the locations below relative to them (I would vote for the latter, as the blockbook command is very long).

Reply:
This is needed for the working directory of blocbook and bitcoind as it expects to be launched with systemd, which sets this directory. This is the solution not to use systemd in docker as that is something we do not want to have.

Reply:
Thanks for the explanation. Cannot still the commands below be made shorter by making them relative, when we are already in that directory?

`./bin/bitcoind -datadir=./backend -conf=./bitcoin_regtest.conf &`

Applies maybe more for the second very long command

Reply:
It's not all in the same directory as the working one. I could shorten it a bit but not that much.

Reply:
I see, both commands contain `/opt/coins/data/` which we are not in. We can certainly live without it, but I would welcome the shortening, if it does not have any disadvantages

Reply:
Tried it with relative commands but it wasn't working.

Reply:
@grdddj's points are good but I think we'll survive if it is causing troubles. Resolving.